packages-web/image-viewer/package.json

```diff
@@ -9,17 +9,17 @@
         "url": "https://github.com/mendix/widgets-resources.git"
     },
     "scripts": {
-        "start": "utils-react-widgets start",
-        "dev": "utils-react-widgets dev",
+        "start": ".\"/node_modules/.bin/utils-react-widgets\" start",
+        "dev": ".\"/node_modules/.bin/utils-react-widgets\" dev",
         "format": "prettier --config \"../../prettier.config.js\" --write \"{src,test}/**/*.{js,jsx,ts,tsx}\"",
         "test": "npm run test:unit",
         "test:unit": "jest --config ../../scripts/test/jest.web.config.js",
-        "test:e2e": "../../node_modules/.bin/cross-env URL=https://imageviewer.mxapps.io/ utils-react-widgets test:e2e",
-        "test:e2e:dev": "../../node_modules/.bin/cross-env DEBUG=true utils-react-widgets test:e2e",
+        "test:e2e": "..\"/../node_modules/.bin/cross-env\" URL=https://imageviewer.mxapps.io/ .\"/node_modules/.bin/utils-react-widgets\" test:e2e",
+        "test:e2e:dev": "..\"/../node_modules/.bin/cross-env\" DEBUG=true .\"/node_modules/.bin/utils-react-widgets\" test:e2e",
         "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
         "lint:fix": "npm run lint -- --fix",
-        "build": "utils-react-widgets build",
-        "release": "utils-react-widgets release",
+        "build": ".\"/node_modules/.bin/utils-react-widgets\" build",
+        "release": ".\"/node_modules/.bin/utils-react-widgets\" release",
         "svncheckout": "ts-node --project ../../scripts/tsconfig.json ../../scripts/svn/CheckoutProject.ts"
```

Comment:
and this one

Reply:
Again, this is just applicable for binaries

Reply:
Done